swim/member.go

```diff
@@ -50,11 +50,11 @@ type Member struct {
 }
 
 // suspect interface
-func (m Member) address() string {
+func (m *Member) address() string {
```

Comment:
```
./swim/member.go:53: address passes Lock by value: swim.Member
```

I believe calling `m.address()` previously would result in `m` being copied. `m` has a mutex on it, so `go vet` calls out this behaviour as potentially harmful.

Correct me if I'm wrong, but it seems to be convention in Golang to pass by reference unless copying is necessary. For this reason, we generally use pointer receivers by default.

I am unaware if the previous code was intentional or not. I can't see any reason why these values would need to be copied.